http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -16,6 +16,8 @@ tags:
     description: Persistent Disks API.
   - name: apps
     description: Experimental App API
+  - name: appsV2
+    description: Apps API v2. Support for Google Dataproc, Google Compute Engine, and Azure.
```

Comment:
Minor nit -- maybe change this text to
```
Apps API v2. Support for Google Kubernetes Engine and Azure Kubernetes Service.
```

(since the apps stuff is all kubernetes.)

Also technically v2 create/delete isn't supported as of this PR (the pubsub messages don't do anything yet ;)). I think it's still fine to add the routes to Swagger, maybe just add `Under active developement.` or something.